src/breadcrumbs/breadcrumbs.scss

```
@@ -28,57 +28,64 @@
   // #region Component overrides
   > a {
     @include iui-anchor;
-    max-width: 24ch;
-    margin: 0 $iui-s;
+    padding: 0 $iui-s;
     overflow: hidden;
     white-space: nowrap;
     text-overflow: ellipsis;
+
+    &:focus,
+    &:focus-visible {
```

`:focus-visible` is unnecessary here and will break the whole selector where it's not supported. https://css-tricks.com/one-invalid-pseudo-selector-equals-an-entire-ignored-selector/

```suggestion
    &:focus {
```

Same thing in the other instance below at line 71

---

Yikes, good catch, fixed.